dts/bindings/sensor/hoperf,th02.yaml

```diff
@@ -0,0 +1,11 @@
+# Copyright (c) 2019, Linaro Limited
+# SPDX-License-Identifier: Apache-2.0
+
+title: TH02 Temperature and Humidity sensor
+
+description: |
+    This is a representation of TH02 Temperature and Humidity sensor
```

Comment:
Could replace with

    TH02 Temperature and Humidity sensor. See datasheet at...

Or whatever other information might be useful.

Would be nice if no more "this is a (base) representation of..." could be added. :)

Reply:
Fixed.